### ISSUE TITLE
[feature]: dark mode for the website

### ISSUE THREAD

**Issue by Swpn0neel** [keploy/keploy#2039]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

A dark mode for the website will be great. Since Tailwind is being used and thus it will be easy to switch to dark mode.

### Why should this be worked on?

Enhanced user experience

### Repository

website

**Comment by DevHrithik** [comment]
@Swpn0neel I am interested in working on this. Also, is there a specific colour palette to work with?

**Comment by HasanZaigam** [comment]
@Swpn0neel I am interested in working on this issue.

**Comment by Adity20** [comment]
I am interested to work on this @Swpn0neel. I have previously worked on integrating the dark theme to JSON Schema website.

**Comment by parthksingh1** [comment]
@Swpn0neel @Sonichigo Please assign this issue to me. I will implement dark mode by updating the Tailwind CSS configuration and ensuring the changes are responsive and accessible.

**Comment by Swpn0neel** [comment]
@HasanZaigam @Adity20 and @parthksingh1 please go ahead, and collaborate with each other!

**Comment by Sonichigo** [comment]
> @Swpn0neel I am interested in working on this. Also, is there a specific colour palette to work with?

Hey @DevHrithik, here's the color pallette - https://github.com/keploy/website?tab=readme-ov-file#brand-color-palette

**Comment by HasanZaigam** [comment]
thanks @Swpn0neel for assigning

**Comment by HasanZaigam** [comment]
@Sonichigo  @Adity20  Have you guys made any changes in this repository?

**Comment by Adity20** [comment]
I already texted you on slack it's been two days @HasanZaigam

**Comment by yaten2302** [comment]
Hey @Adity20 , @HasanZaigam , @parthksingh1 , are you all working on this issue, may I also collaborate with you all on this issue? @Sonichigo @Swpn0neel

**Comment by techmannih** [comment]
@Sonichigo @Swpn0neel there has been no activity for the last 2 months. May I start working on this issue?

**Comment by Swpn0neel** [comment]
Go ahead @techmannih

**Comment by techmannih** [comment]
@Sonichigo @Swpn0neel could you suggest any background you prefer?

**Comment by Sonichigo** [comment]
@techmannih you can take a look at docs website for the themes and references.

**Comment by techmannih** [comment]
@Sonichigo okay thank you! got it.

**Comment by Sonichigo** [comment]
hi @techmannih any progress to share?

**Comment by techmannih** [comment]
Yes, I was busy with another issue, but I am working on this now.

**Comment by techmannih** [comment]
@Sonichigo , please check the dark mode theme and colors 

https://github.com/user-attachments/assets/4da6ac43-1d7a-40a4-b575-717d2856c28d

**Comment by charankamarapu** [comment]
Any update on this issue @Sonichigo ..?

**Comment by Sonichigo** [comment]
Since this issue got stale and backlog, this can be closed @charankamarapu

**Comment by Aviral0702** [comment]
Hi, is this issue still available? I will work on it.
